content/de/docs/concepts/containers/images.md

```diff
@@ -326,15 +326,15 @@ Es gibt eine Anzahl an Lösungen um eigene Registries zu konfigurieren, hier sin
      - Funktioniert besser mit Cluster - Autoskalierung als mit manueller Knotenkonfiguration.
    - Auf einem Cluster bei dem die Knotenkonfiguration ungünstig ist können `imagePullSecrets` genutzt werden.
 3. Cluster mit proprieritären Images, mit einigen Images die eine erweiterte Zugriffskontrolle erfordern.
-   - Stellen sie sicher das [AlwaysPullImages admission controller](/docs/reference/access-authn-authz/admission-controllers/#alwayspullimages) aktiv ist, anderenfalls können alle Pods potenziell Zugriff auf alle Images haben.
+   - Stellen sie sicher, dass [AlwaysPullImages admission controller](/docs/reference/access-authn-authz/admission-controllers/#alwayspullimages) aktiv ist, anderenfalls können alle Pods potenziell Zugriff auf alle Images haben.
    - Verschieben sie sensitive Daten in eine "Secret" Ressource statt sie im Image abzulegen.
 4. Ein mandantenfähiger Cluster in dem jeder Mandant eine eigene private Registry benötigt.
-   - Stellen sie dicher das [AlwaysPullImages admission controller](/docs/reference/access-authn-authz/admission-controllers/#alwayspullimages) aktiv ist, anderenfalls können alle Pods potenziell Zugriff auf alle Images haben.
+   - Stellen sie sicher, dass [AlwaysPullImages admission controller](/docs/reference/access-authn-authz/admission-controllers/#alwayspullimages) aktiv ist, anderenfalls können alle Pods potenziell Zugriff auf alle Images haben.
    - Nutzen sie eine private Registry die eine Authorisierung erfordert. 
-   - Generieren die Registry - Zugriffsdaten für jeden Mandanten, abgelegt in einem Secret das in jedem Mandanten - Namespace vorhanden ist.
+   - Generieren sie Registry - Zugriffsdaten für jeden Mandanten, abgelegt in einem Secret das in jedem Mandanten - Namespace vorhanden ist.
```

Comment:
I think this was correct initially. @mkorbi what is you opinion here?

Reply:
Think ether both missing something and should be: "Generieren sie die Registry ..."